crates/shared/src/paraswap_api.rs

```diff
@@ -374,7 +373,7 @@ mod tests {
             exclude_dexs: None,
         };
 
-        let url = price_query.into_url("cowswap");
+        let url = price_query.into_url("https://apiv5.paraswap.io", "cowswap");
```

Comment:
In case of `apiv6`, maybe it makes sense to have a:

```
const DEFAULT_URL: &str = "https://apiv5.paraswap.io";
```

In the ParaSwap module?